### PR TITLE
DDF-1698 Fix catalog:removeall command

### PR DESCRIPTION
**Pull request by harrison-tarr**
[DDF-1698](https://codice.atlassian.net/browse/DDF-1698) introduced a bug where when this new feature (catalog-plugin-metacard-validation) is on, `catalog:removeall` fails to remove all records. This was caused by a disparity between the expected batch size and the actual page size.

Most of the changes in this PR are formatting changes. The actual code changes are:
RemoveAllCommand:
line 37
line 292-309
line 332-349

DynamicSchemaResolver:
line 195-198

MetacardValidityCheckerPlugin:
line 60-63

@AzGoalie @brendan-hofmann @ahoffer

### CODE REVIEW

**Comment by review-ninja** [comment]
<a href="https://app.review.ninja/codice/ddf/pull/529" target="_blank">ReviewNinja

File: catalog/core/catalog-core-solr/src/main/java/ddf/catalog/source/solr/DynamicSchemaResolver.java [hunk 1]
```diff
@@ -184,6 +192,10 @@ public void addFields(Metacard metacard, SolrInputDocument solrInputDocument)
             if (metacard.getAttribute(ad.getName()) != null) {
                 List<Serializable> attributeValues = metacard.getAttribute(ad.getName())
                         .getValues();
+                if (ad.getType()
```
Comment (brendan-hofmann) [inline catalog/core/catalog-core-solr/src/main/java/ddf/catalog/source/solr/DynamicSchemaResolver.java]: Why is this here, and why is not already handled by `addToFieldsCache`? It seems redundant.

Comment (harrison-tarr) [inline catalog/core/catalog-core-solr/src/main/java/ddf/catalog/source/solr/DynamicSchemaResolver.java]: I addressed this in the commit message. 

> Dynamic schema resolver had trouble finding validation-warnings and
validation-errors fields in the fieldsCache. Now it doesn't. However
this may have been a fluke.

After I changed this and rebuilt it worked, but I'm not sure if it working was caused by this change or a different configuration. I think this portion may need more investigation.

There are two problems that make records not get removed. 1. The paging and batchsize didn't match up. 2. Before with `catalog-plugin-metacard-validation` on and now by default the query used to delete metacards will search for `validation-errors` and `validation-warnings`

The first problem _is_ fixed by the first commit and the second commit is _intended_ to fix the second.

File: catalog/core/catalog-core-commands/src/main/java/org/codice/ddf/commands/catalog/RemoveAllCommand.java [hunk 2]
```diff
@@ -280,10 +289,29 @@ private String getTotalAmount(long hits) {
     private QueryRequest getIntendedQuery(FilterBuilder filterBuilder, boolean isRequestForTotal)
             throws InterruptedException {
 
-        Filter filter = filterBuilder.attribute(Metacard.ID).is().like().text(WILDCARD);
+        Filter filter = filterBuilder.allOf(filterBuilder.attribute(Metacard.ID)
```
Comment (brendan-hofmann) [inline catalog/core/catalog-core-commands/src/main/java/org/codice/ddf/commands/catalog/RemoveAllCommand.java]: It looks like there's some duplication you could eliminate in these two filters.

**Comment by spearskw** [comment]
:+1: PRBS completed successfully

**Comment by kcwire** [comment]
Need better title - "bugfix" doesn't tell me anything.

**Comment by spearskw** [comment]
Also manually tested and the bugfix works.

**Comment by NGoss** [comment]
PRBS successfully completed
